desk/app/activity.hoon

```diff
@@ -861,7 +865,14 @@
   ::  with new reads, update our index and summary
   =.  cor  (refresh-index source index)
   =/  new=(unit activity-summary:a)  (~(get by activity) source)
-  ~?  !=(old new)  "%sync-reads: WARNING old and new summaries differ {<old>} {<new>}"
+  =/  old-sum  ?~(old ~ %=(u.old reads ~))
+  =/  new-sum  ?~(new ~ %=(u.new reads ~))
```

Comment:
Might be the first time I've seen short-form `%=` instead of the sugar. (^:

```suggestion
  =/  old-sum  ?~(old ~ u.old(reads ~))
  =/  new-sum  ?~(new ~ u.new(reads ~))
```

You might also consider moving these to be inline in the `~&` below, since that's the only place they're used. In general I'd be weary of introducing stuff into the subject ahead/outside of the one branch of the code where it gets used.